src/cpx_constrs.jl

```diff
@@ -156,6 +166,21 @@ function get_constr_senses(model::Model)
     return senses
 end
 
+function chg_sense!(model::Model, indices::IVec, senses::CVec)
```

Comment:
Style: I think if we are wrapping new CPLEX functions, we should use exactly the CPLEX name. So refactor `chg_sense!` to `chgsense`.

Reply:
I only added few functions, so I tried to keep the style consistent with the rest of the wrapper functions. Plus I think if we use underscores everywhere else in julia code, it's better to keep those underscores. I'll open an issue to see what the others think of this.